vite.config.js

```diff
@@ -0,0 +1,18 @@
+import { resolve } from 'path';
+
+// Consult https://vitejs.dev/config/ to learn about these options
+/** @type {import("vite").UserConfig } */
+const config = {
+    resolve: {
+        alias: {
+            $components: resolve('src/components'),
+            $app: resolve('.svelte/assets/runtime/app')
```

Comment:
I believe this line is not necessary - SvelteKit already does it for us internally.